backend/app.py

```diff
@@ -114,14 +113,118 @@ def verify():
+
+    # salesforce query of each completed outcome # in associate pod, based on the email and name
+    soql = "SELECT {} FROM Associate_POD_Map__c".format(','.join(Associate_field_names))
+    Associate_sf_result = sf.query(format_soql((soql + " WHERE (Contact__r.email = {email_value} AND Contact__r.name={full_name})"), email_value=email, full_name=name))
+
+    # calculate Trainee total 
+    Associate_total_count = 0; #create new value in sf_result dict that will store field's total outcomes 
+    for field in Associate_field_names:
+        field_type = field[3:6].upper()
+        for name_and_label in Associate_field_names_and_labels:
+            if "_Outcome_" + field_type in name_and_label[0]: 
+                Associate_total_count += 1
+
+    # transform into a python dictionary
+    vars(Associate_sf_result)
+
+    # calculate *Trainee* outcomes based on all related fields
+    Associate_outcome_sum = 0
+    for outcome in Associate_field_names:
+        Associate_outcome_sum += Associate_sf_result['records'][0][outcome]
+
+  
+# Partner Pod
+    desc = sf.Partner_POD_Map__c.describe()
+    Partner_field_names_and_labels = [(field['name'], field['label']) for field in desc['fields']]
+    Partner_filtered_field_names = [field for field in Partner_field_names_and_labels if "Completed__c" in field[0]]
+    Partner_field_names = [field[0] for field in Partner_filtered_field_names]
+
+    # calculate Trainee total 
+    Partner_total_count = 0; #create new value in sf_result dict that will store field's total outcomes 
+    for field in Partner_field_names:
+        field_type = field[3:6].upper()
+        for name_and_label in Partner_field_names_and_labels:
+            if "_Outcome_" + field_type in name_and_label[0]: 
+                Partner_total_count += 1
+     
+    # salesforce query of each completed outcome # in trainee pod, based on the email and name
+    soql = "SELECT {} FROM Partner_POD_Map__c".format(','.join(Partner_field_names))
+    Partner_sf_result = sf.query(format_soql((soql + " WHERE (Contact__r.email = {email_value} AND Contact__r.name={full_name})"), email_value=email, full_name=name))
+  
+    # print('_________________________________________________')
+    # print(Partner_sf_result)
+
+    # transform into a python dictionary
+    # vars(Partner_sf_result)
+
+    # calculate *Partner* outcomes based on all related fields
+    # Partner_outcome_sum = 0
+    # for outcome in Partner_field_names:
+    #     Partner_outcome_sum += Partner_sf_result['records'][0][outcome]
+
+    # print(Partner_outcome_sum)
+    
+    # TODO:  Partner Pod
+    sum_pod_outcome = {
+        'Trainee_complete': Trainee_outcome_sum,
+        'Trainee_total': Trainee_total_count,
+        'Associate_complete': Associate_outcome_sum,
+        'Associate_total': Associate_total_count,
+        # 'Partner_complete': Partner_outcome_sum,
+        'Partner_total': Partner_total_count
+    }    
```

Comment:
Nice job cleaning up the data and only passing back to the frontend what is necessary